cmd/migrations/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/go-devs-ua/octagon/cfg"
+	"github.com/go-devs-ua/octagon/lgr"
+	_ "github.com/lib/pq"
+
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+const (
+	up   = "up"
+	down = "down"
+)
+
+func main() {
+	logger := lgr.New()
+	// defer logger.Flush()
```

Comment:
Commented code is not allowed in main branch. Please remove it.

Reply:
Fixed.